package.json

```diff
@@ -1,29 +1,29 @@
 {
-  "name": "bitmovin-player-analytics-comscore",
+  "name": "@bitmovin/player-integration-comscore",
   "version": "1.0.0",
   "description": "ComScore analytics integration for the Bitmovin Player",
   "repository": {
     "type": "git",
     "url": "https://github.com/bitmovin/bitmovin-player-analytics-comscore.git"
   },
-  "main": "bitmovinplayer-analytics-comscore.js",
-  "module": "lib/index.js",
-  "es2015": "lib-esm/index.js",
+  "main": "./dist/bitmovin-player-analytics-comscore.js",
```

Comment:
Naming of files doesn't fit the module name.

Reply:
must they fit?

Reply:
Would be nicer. This way seems chaotic.

Reply:
I see but I thought that i don't change it as this is already published as it is. Not sure if this can be considered as API change.